scripts/render.py

```diff
@@ -42,6 +44,7 @@ def _render_trajectory_video(
     rendered_resolution_scaling_factor: float = 1.0,
     seconds: float = 5.0,
     output_format: Literal["images", "video"] = "video",
+    camera_type: str = "",
```

Comment:
Can you make this a Literal and add it to the doc string below.

Reply:
I discovered the CameraType enum and switched to that instead.